src/Nethermind/Nethermind.Flashbots/IFlashbotsConfig.cs

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Config;
+
+namespace Nethermind.Flashbots;
+
+public interface IFlashbotsConfig : IConfig
+{
+    [ConfigItem(Description = "Whether to enable the Flashbots endpoints.", DefaultValue = "false")]
+    bool Enabled { get; set; }
+
+    [ConfigItem(Description = "If set to true, proposer payment is calculated as a balance difference of the fee recipient", DefaultValue = "false")]
```

Comment:
We should keep the wording the same as this goes to the docs:

```suggestion
    [ConfigItem(Description = "Whether to calculate the proposer payment as a balance difference of the fee recipient.", DefaultValue = "false")]
```